repose-aggregator/tests/performance-tests/Jenkinsfile

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env groovy
+
+properties([
+        buildDiscarder(logRotator(numToKeepStr: '14')),
+        pipelineTriggers([cron('H H(0-7) * * *')]),
+        parameters([string(
+                defaultValue: "",
+                description: "Optional - Extra options to add to the end of the ansible-playbook command.",
+                name: "extraOptions")])
+])
+
+stage("Performance Test") {
+    // current list of performance tests to run (with the optional scripting language if applicable)
+    def perfTestsWithScriptLang = [
```

Comment:
ohh, i thought we had scoped it in for some reason. Also i suspect solving this will involve separating out the scripting filter tests anyway